images/connect-operator/hooks/operator.sh

```diff
@@ -61,13 +61,13 @@ function delete_connector() {
     curl_user_opt="--user $(get_cc_kafka_cluster_connect_user_arg)"
   fi
 
-  trap 'rm -f "$tmpfile"' EXIT
+  trap 'rm -f "$tmpfile"' RETURN
   local tmpfile=$(mktemp) || exit 1
   echo $config > $tmpfile
 
   local template_command="jq -c -n $JQ_ARGS_FROM_CONFIG_FILE -f $tmpfile"
 
-  local desired_connector_config=$(eval $template_command | jq -c '.config')
+  local desired_connector_config=$(eval $template_command)
```

Comment:
@krbcmt thank you for the PR.   Question about this change... the pipe to `jq` to pull out the `.config` entry in the resulting configuration is intentional, and the next line to pull out the connector name expects the config to be specific.  Did you test these changes by chance to ensure this change is valid?  Thanks again

Reply:
Given the configMap:
```
kind: ConfigMap
...
data:
  a-connector-definition: |
    {
      "name": "some-connector-name",
      "config": {
        ...
      }
```
I've been under the impression that the `.config.name` field may not always exist (connector examples I've seen online and configs my dev teams has created), but `.name` always will so that should be used instead. Line 105 of the original version of this file seems to make the same assumption. My local testing indicates this is a valid change.